app/src/main/java/org/p2p/wallet/App.kt

```diff
@@ -108,4 +110,18 @@ class App : Application() {
     private fun setupTimber() {
         if (BuildConfig.DEBUG) Timber.plant(Timber.DebugTree())
     }
+
+    private fun createSolanjLogger(): SolanjLogger = object : SolanjLogger {
```

Comment:
Maybe instead of creating class we make a object(static class) SolanaLogger.d("message") i think create a singleton class is redundant

Reply:
good idea
done